visualization/src/pcl_visualizer.cpp

```diff
@@ -173,10 +173,10 @@ pcl::visualization::details::fillCells(std::vector<int>& lookup, const std::vect
   }
 #endif
 
-  const auto idx = vertices.size() + std::accumulate(vertices.begin(), vertices.end(), 0,
+  const auto idx = static_cast<vtkIdType>(vertices.size()) + std::accumulate(vertices.begin(), vertices.end(), 0,
```

Comment:
`std::accumulate(vertices.begin(), vertices.end(), 0,` still might overflow.

https://godbolt.org/z/8W9Wdf

Reply:
So we'll need static cast both places? If I don't do static_cast on `vertices.size()` idx gets resolved to unsigned long long.

Reply:
Just one. On the zero. Everywhere else, the larger one is chosen by the rules of promotion.